BitmovinYoSpaceModule/Classes/BitmovinYospacePlayer.swift

```diff
@@ -916,16 +924,12 @@ extension BitmovinYospacePlayer: PlayerListener {
         }
     }
 
-    public func onWarning(_ event: YospaceWarningEvent, player: Player) {
-        for listener: PlayerListener in listeners {
-            listener.onEvent?(event, player: player)
-        }
+    public func onPlayerError(_ event: PlayerErrorEvent, player: Player) {
+        listeners.forEach { $0.onPlayerError?(event, player: player) }
     }
 
-    public func onError(_ event: YospaceErrorEvent, player: Player) {
```

Comment:
The onError listener method no longer exists as we separated them into `Player-` and `SourceErrors` in v3. Therefore it's no longer possible to also use those methods for emitting `YospaceErrorEvent` as the method signature no longer matches. (This might only worked by accident earlier due hos the ObjC runtime works - I don't expect that this ever worked since the v3 update)

I fixed this by introducing dedicated methods to emit those events.